rpc/ipc_unix.go

```diff
@@ -32,8 +32,8 @@ import (
 // ipcListen will create a Unix socket on the given endpoint.
 func ipcListen(endpoint string) (net.Listener, error) {
 	// account for null-terminator too
-	if len(endpoint)+1 > int(max_path_size) {
-		log.Warn(fmt.Sprintf("The ipc endpoint is longer than %d characters. ", max_path_size-1),
+	if len(endpoint)+1 > maxPathSize {
```

Comment:
If we want to make things simpler, why not just put the constant in this file and get rid of `constants.go` too?
We can just  
```suggestion
	//  On Linux, sun_path is 108 bytes in size
	// see http://man7.org/linux/man-pages/man7/unix.7.html
	var maxPathSize = int(108)
	if len(endpoint)+1 > maxPathSize {
```